Homework 19

YuriiLyndiuk opened the pull request:
Verify that the confirmation notification displayed has the text, "Deleted playlist {playlist name}".

TestPro-Support committed:
Added L16 examples and updated webdrivermanager

muhammadTestPro committed:
Commit Test

muhammadTestPro committed:
17Lesson Complete

muhammadTestPro committed:
lesson18 - Complete

muhammadTestPro committed:
Lesson 19 Complete

YuriiLyndiuk committed:
Homework-19

 Verify that the  notification has the text, "Deleted playlist {playlist name}".

Berichicko commented:
You got this problem, please correct it.

![image](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/1d6d6f7d-b808-47f2-b3a7-0111eff513b4)

HINT

**Step 1**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

For example

Add 

`public  WebDriverWait wait = null;`


```
    public WebDriver driver;
    public  WebDriverWait wait = null;


    @BeforeSuite
    static void setupClass() {
        WebDriverManager.chromedriver().setup();
    }


    @BeforeMethod
    @Parameters({"BaseURL"})
    public void launchBrowser(String baseURL){
        ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        options.setExperimentalOption("excludeSwitches", new String[]{"enable-automation"});
        driver = new ChromeDriver(options);
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        navigateToPage(baseURL);
    }
```

![image](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/5cb690c7-2277-4b79-9f8a-f461bbbd29dc)

**Step 2**

Add `wait.until(ExpectedConditions.visibilityOfElementLocated`

![image](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/17582742-de87-41dc-a4de-1a95efa20d0b)

YuriiLyndiuk committed:
Homework-19

Verify that the confirmation notification displayed has the text, "Deleted playlist {playlist name}".

YuriiLyndiuk commented:
Explicit Wait command its Homework 20. 
I used the Thread.sleep method for this task.
Please check.
Thank you!
